src/main/java/org/jfrog/hudson/generic/ArtifactoryGenericConfigurator.java

```diff
@@ -100,6 +100,8 @@ public class ArtifactoryGenericConfigurator extends BuildWrapper implements Depl
     @Deprecated
     private final String matrixParams = null;
 
+    private String project;
```

Comment:
Let's add the following comment for this new member:
// JFrog project key

The same goes for this new member in the `ArtifactoryGradleConfigurator` class.